lib/rex/proto/mssql/client_mixin.rb

```diff
@@ -58,7 +58,7 @@ def mssql_print_reply(info)
         tbl << row
       end
 
-      print_line(tbl.to_s)
+      print_line(tbl.to_s.gsub("\n\n\n","\n"))
```

Comment:
This looks like a hack; It looks like the extra whitespace is caused by the Header being blank

```diff
      tbl = Rex::Text::Table.new(
        'Indent'    => 1,
-        'Header'    => "",
+        'Header'    => "Response",
        'Columns'   => info[:colnames],
        'SortIndex' => -1
      )

```

Which would give a cleaner output without the whitespace that I think you were trying to remove

```
[*] 192.168.123.136:1433  - SQL Query: select @@version;
[*] 192.168.123.136:1433  - Row Count: 1 (Status: 16 Command: 193)
Response
========

 NULL
 ----
 Microsoft SQL Server 2022 (RTM) - 16.0.1000.6 (X64) 
	Oct  8 2022 05:58:25 
	Copyright (C) 2022 Microsoft Corporation
	Express Edition (64-bit) on Windows Server 2022 Standard Evaluation 10.0 <X64> (Build 20348: ) (Hypervisor)

```